### PR TITLE
New fullstack dev

### PR DESCRIPTION
**Pull request by jrodom**
Make our full stack dev posting more robust and complete.

### CODE REVIEW

**Comment by leemunroe** [comment]
:shipit:

File: fullstack.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+###Responsibilities
+* Build and develop new user and support-facing features in the Mailgun control panel. 
+* Develop the necessary backend REST services to support new features that will be exposed in our control panel. 
+* Partner with product designer to implement mockups and prototypes.  Help refine designs to achieve the best user experience. 
+
+###Requirements 
+* BS in Computer Science or related area, or relevant work experience.
+* 2-3 years experience working with cutting-edge JavaScript frameworks (AngularJS, Ember, etc.) 
+* Experience with CSS preprocessors (Sass, LESS, etc.)
+* Experience building RESTful web services, preferably in Python.
```
Comment (jessespears) [inline fullstack.md]: Maybe add Go as an alternate programming language.  All our new services are written in go.

**Comment by jessespears** [comment]
other than my comment, lgtm

**Comment by r0mant** [comment]
Do you think we should add "being on-call" requirement? All our devs are on-call once/twice a week.

**Comment by jessespears** [comment]
It might imply much more midnight wakeups than we actually experience,
especially since we have European coverage right now.  Maybe "occasional
daytime on-call" or something to that effect.

File: fullstack.md (hunk 1 above)
Comment (jrodom) [inline fullstack.md]: Good idea, will add this.

**Comment by jrodom** [comment]
I think adding an on-call note is good.  While its only occasionally an issue, it's good to call it out.

**Pull request merged**